docker/advanced_settings.md

```diff
@@ -335,111 +295,43 @@ The predict endpoint will return a prediction response in JSON. It will look som
 }
 ```
 
-
 ## Description of Config File Settings
 
-**For mms_app_cpu.conf:**
+**For mms_config.properties:**
 
-The system settings are stored in [mms_app_cpu.conf](mms_app_cpu.conf). You can modify these settings to use different models, or to apply other customized settings. The default settings were optimized for a c5.2xlarge instance.
+The system settings are stored in [mms_config.properties](mms_config.properties). You can modify these settings to use different models, or to apply other customized settings. 
```

Comment:
We can link to [mms_config.properties](../docker/config.properties)

Reply:
Thats what the above does. It links to the local directory.

Reply:
I mean config.properties.

We should not rename it to mms_config.properties. This file is used in DockerFile and MMS will only pickup config.properties in working folder, not mms_config.properties.